run.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+sbt -Dplay.http.router=testOnlyDoNotUseInAppConf.Routes "run 6000"
```

Comment:
I'll run the tests quickly, this looks good though 🚀 

```suggestion
sbt -Dplay.http.router=testOnlyDoNotUseInAppConf.Routes "run 7750"
```